
<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<title>IDEAS | Feed</title>
		<style>
			@import url("https://fonts.googleapis.com/css2?family=Nanum+Myeongjo&display=swap");

			body {
				font-family: "Nanum Myeongjo", serif;
				line-height: 1.7;
				max-width: 600px;
				margin: 50px auto 50px;
				padding: 0 12px 0;
				height: 100%;
			}

			li {
				padding-bottom: 16px;
			}
			.container {
				overflow: hidden;
			  }
			  
			  .filterDiv {
				float: left;
				width: 100%;
				text-align: left;
				margin: 2px;
				display: none; /* Hidden by default */
			  }
			  
			  /* The "show" class is added to the filtered elements */
			  .show {
				display: block;
			  }
			  
			  /* Style the buttons */
			  .btn {
				border: none;
				outline: none;
				padding: 12px 16px;
				background-color: #f1f1f1;
				cursor: pointer;
			  }
			  
			  /* Add a light grey background on mouse-over */
			  .btn:hover {
				background-color: #ddd;
			  }
			  
			  /* Add a dark background to the active button */
			  .btn.active {
				background-color: #666;
				color: white;
			  }
		</style>
		<script>
		filterSelection("all")
function filterSelection(c) {
  var x, i;
  x = document.getElementsByClassName("filterDiv");
  if (c == "all") c = "";
  // Add the "show" class (display:block) to the filtered elements, and remove the "show" class from the elements that are not selected
  for (i = 0; i < x.length; i++) {
    w3RemoveClass(x[i], "show");
    if (x[i].className.indexOf(c) > -1) w3AddClass(x[i], "show");
  }
}

// Show filtered elements
function w3AddClass(element, name) {
  var i, arr1, arr2;
  arr1 = element.className.split(" ");
  arr2 = name.split(" ");
  for (i = 0; i < arr2.length; i++) {
    if (arr1.indexOf(arr2[i]) == -1) {
      element.className += " " + arr2[i];
    }
  }
}

// Hide elements that are not selected
function w3RemoveClass(element, name) {
  var i, arr1, arr2;
  arr1 = element.className.split(" ");
  arr2 = name.split(" ");
  for (i = 0; i < arr2.length; i++) {
    while (arr1.indexOf(arr2[i]) > -1) {
      arr1.splice(arr1.indexOf(arr2[i]), 1);
    }
  }
  element.className = arr1.join(" ");
}

// Add active class to the current control button (highlight it)
var btnContainer = document.getElementById("myBtnContainer");
var btns = btnContainer.getElementsByClassName("btn");
for (var i = 0; i < btns.length; i++) {
  btns[i].addEventListener("click", function() {
    var current = document.getElementsByClassName("active");
    current[0].className = current[0].className.replace(" active", "");
    this.className += " active";
  });
}
		</script>
	</head>
	<body>
	<P></P><cite><a href="https://wakeforestid.com/">home</a></cite>
		<h1>Latest Literature</h1>
		<div id="myBtnContainer">
		<button class="btn active" id="al2l" onclick="filterSelection('all')"> Show all</button>
		<button class="btn" id="all" onclick="filterSelection('amr')"> AMR</button>
		<button class="btn" onclick="filterSelection('fungal')"> Fungal</button>
		<button class="btn" onclick="filterSelection('vector')"> Vector</button>
		<button class="btn" onclick="filterSelection('bacteria')"> Bacterial</button>
		<button class="btn" onclick="filterSelection('std')"> STD</button>
		<button class="btn" onclick="filterSelection('surveillance')"> Surveillance</button>
		<button class="btn" onclick="filterSelection('modelling')"> Modelling</button>
		<button class="btn" onclick="filterSelection('viral')"> Viral</button>
		<button class="btn" onclick="filterSelection('omics')"> Omics and Biomarkers</button>
		</div>

		<div class="container">
		<ol>
			<div class="filterDiv all save std viral omics">
			  <li><a href="https://www.jwatch.org/na55667/2023/01/10/vaginal-microbiome-and-hiv-susceptibility">The Vaginal Microbiome and HIV Susceptibility</a><p>In young South African women, polymicrobial-predominant vaginal microbiomes were associated with higher risk for acquiring HIV compared with Lactobacillus crispatus–predominant microbiomes.</p> (www.jwatch.org)</li>
			  </div>
			<div class="filterDiv all save surveillance viral">
			  <li><a href="https://onlinelibrary.wiley.com/doi/10.1111/irv.13089?af=R">Developing a sampling methodology for timely reporting of population‐based COVID‐19‐associated hospitalization surveillance in the United States, COVID‐NET 2020–2021</a><p>Abstract 
Background 
The COVID-19-Associated Hospitalization Surveillance Network (COVID-NET) required a sampling methodology that allowed for production of timely population-based clinical estimates to inform the ongoing US COVID-19 pandemic response. 
Methods 
We developed a flexible sampling approach that considered reporting delays, differential hospitalized case burden across surveillance sites, and changing geographic and demographic trends over time. We incorporated weighting methods to adjust for the probability of selection and non-response, and to calibrate the sampled case distribution to the population distribution on demographics. We additionally developed procedures for variance estimation. 
Results 
Between March 2020 and June 2021, 19,293 (10.4%) of all adult hospitalized cases were sampled for chart abstraction. Variance estimates for select variables of interest were within desired ranges. 
Conclusions 
COVID-NET's sampling methodology allowed for reporting of robust and timely, population-based data on the clinical epidemiology of COVID-19-associated hospitalizations and evolving trends over time, while attempting to reduce data collection burden on surveillance sites. Such methods may provide a general framework for other surveillance systems needing to quickly and efficiently collect and disseminate data for public health action.</p> (onlinelibrary.wiley.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria">
			  <li><a href="https://dx.doi.org/10.1017/S0950268822000899?rft_dat=source%3Ddrss">Non-tuberculous mycobacteria: clinical and laboratory characterisation (2009 and 2019)</a><p>  
A cross-sectional and retrospective study of patients with Mycobacterium spp. in a Portuguese tertiary hospital, in 2009 and 2019, was performed to understand better the rise in isolations of nontuberculous mycobacteria (NTM). The number of patients with positive samples for Mycobacterium spp. grew from 56 in 2009 to 83 in 2019. The proportion of NTM rose from 39.3% to 49.4% (P = 0.240), with Mycobacterium avium complex being more frequent in 2009 and Mycobacterium gordonae in 2019, and Mycobacterium tuberculosis complex decreased from 60.7% to 50.6%. Higher age was associated with NTM in both years, and pulmonary disease and immunosuppression were associated with NTM in 2019 (P &lt; 0.05), with weak to moderate correlation (V = 0.231–0.343). The overall rise of NTM, allied to their known capacity to resist antimicrobial therapy, alerts clinicians to the importance of recognising potential risk factors for infection and improving future prevention strategies.  
</p> (dx.doi.org)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling viral">
			  <li><a href="https://onlinelibrary.wiley.com/doi/10.1111/irv.13087?af=R">Investigating confounding in network‐based test‐negative design influenza vaccine effectiveness studies—Experience from the DRIVE project</a><p>Abstract 
Background: Establishing a large study network to conduct influenza vaccine effectiveness (IVE) studies while collecting appropriate variables to account for potential bias is important; the most relevant variables should be prioritized. We explored the impact of potential confounders on IVE in the DRIVE multi-country network of sites conducting test-negative design (TND) studies. 
Methods: We constructed a directed acyclic graph (DAG) to map the relationship between influenza vaccination, medically attended influenza infection, confounders, and other variables. Additionally, we used the Development of Robust and Innovative Vaccines Effectiveness (DRIVE) data from the 2018/2019 and 2019/2020 seasons to explore the effect of covariate adjustment on IVE estimates. The reference model was adjusted for age, sex, calendar time, and season. The covariates studied were presence of at least one, two, or three chronic diseases; presence of six specific chronic diseases; and prior healthcare use. Analyses were conducted by site and subsequently pooled. 
Results: The following variables were included in the DAG: age, sex, time within influenza season and year, health status and comorbidities, study site, health-care-seeking behavior, contact patterns and social precautionary behavior, socioeconomic status, and pre-existing immunity. Across all age groups and settings, only adjustment for lung disease in older adults in the primary care setting resulted in a relative change of the IVE point estimate &gt;10%. 
Conclusion: Our study supports a parsimonious approach to confounder adjustment in TND studies, limited to adjusting for age, sex, and calendar time. Practical implications are that necessitating fewer variables lowers the threshold for enrollment of sites in IVE studies and simplifies the pooling of data from different IVE studies or study networks.</p> (onlinelibrary.wiley.com)</li>
			  </div>
			<div class="filterDiv all save surveillance viral omics">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/2/22-1482_article">Sentinel Surveillance System Implementation and Evaluation for SARS-CoV-2 Genomic Data, Washington, USA, 2020–2021</a><p>H. N. Oltean et al.</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save surveillance viral">
			  <li><a href="https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-022-07967-2">Effects of contact tracing and nucleic acid testing on the COVID-19 outbreak in Zunyi, China: data-driven study using a branching process model</a><p>During October 2021, China experienced localized outbreaks of COVID-19 in many cities. We analyzed the small local outbreak in Zunyi (Guizhou Province), a major city in southwestern China, and modeled the effe...</p> (bmcinfectdis.biomedcentral.com)</li>
			  </div>
			<div class="filterDiv all save viral">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/2/22-0712_article">Longitudinal Analysis of Electronic Health Information to Identify Possible COVID-19 Sequelae</a><p>E. S. Click et al.</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save vector">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/1/ac-2901_article">Up Close with Ticks</a><p>B. Breedlove</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save surveillance vector viral">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/1/21-2295_article">Comprehensive Review of Emergence and Virology of Tickborne Bourbon Virus in the United States</a><p>M. K. Roe et al.</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling viral">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/1/22-1009_article">High SARS-CoV-2 Seroprevalence after Second COVID-19 Wave (October 2020–April 2021), Democratic Republic of the Congo</a><p>Y. Munyeku-Bazitama et al.</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling viral">
			  <li><a href="https://onlinelibrary.wiley.com/doi/10.1111/irv.13082?af=R">Trends in disparities in COVID hospitalizations among community‐dwelling residents of two counties in Connecticut, before and after vaccine introduction, March 2020–September 2021</a><p>Abstract 
Background 
Prior to the introduction of vaccines, COVID-19 hospitalizations of non-institutionalized persons in Connecticut disproportionately affected communities of color and individuals of low socioeconomic status (SES). Whether the magnitude of these disparities changed 7–9 months after vaccine rollout during the Delta wave is not well documented. 
Methods 
All initially hospitalized patients with laboratory-confirmed COVID-19 during July–September 2021 were obtained from the Connecticut COVID-19-Associated Hospitalization Surveillance Network database, including patients' geocoded residential addresses. Census tract measures of poverty and crowding were determined by linking geocoded residential addresses to the 2014–2018 American Community Survey. Age-adjusted incidence and relative rates of COVID-19 hospitalization were calculated and compared with those from July to December 2020. Vaccination levels by age and race/ethnicity at the beginning and end of the study period were obtained from Connecticut's COVID vaccine registry, and age-adjusted average values were determined. 
Results 
There were 708 COVID-19 hospitalizations among community residents of the two counties, July–September 2021. Age-adjusted incidence was the highest among non-Hispanic Blacks and Hispanic/Latinx compared with non-Hispanic Whites (RR 4.10 [95% CI 3.41–4.94] and 3.47 [95% CI 2.89–4.16]). Although RR decreased significantly among Hispanic/Latinx and among the lowest SES groups, it increased among non-Hispanic Blacks (from RR 3.2 [95% CI 2.83–3.32] to RR 4.10). Average age-adjusted vaccination rates among those ≥12 years were the lowest among non-Hispanic Blacks compared with Hispanic/Latinx and non-Hispanic Whites (50.6% vs. 64.7% and 66.6%). 
Conclusions 
Although racial/ethnic and SES disparities in COVID-19 hospitalization have mostly decreased over time, disparities among non-Hispanic Blacks increased, possibly due to differences in vaccination rates.</p> (onlinelibrary.wiley.com)</li>
			  </div>
			<div class="filterDiv all save modelling omics">
			  <li><a href="https://onlinelibrary.wiley.com/doi/10.1002/sim.9652?af=R">Covariate‐specific evaluation of continuous biomarker</a><p>Diagnostic tests usually need to operate at a high sensitivity or specificity level in practice. Accordingly, specificity at the controlled sensitivity, or vice versa, is a clinically sensible performance metric for evaluating continuous biomarkers. Meanwhile, the performance of a biomarker may vary across sub-populations as defined by covariates, and covariate-specific evaluation can be informative. In this article, we develop a novel modeling and estimation method for covariate-specific specificity at a controlled sensitivity level. Unlike existing methods which typically adopt elaborate models of covariate effects over the entire biomarker distribution, our approach models covariate effects locally at a specific sensitivity level of interest. We also extend our proposed model to handle the whole continuum of sensitivities via dynamic regression and derive covariate-specific ROC curves. We provide the variance estimation through bootstrapping. The asymptotic properties are established. We conduct extensive simulation studies to evaluate the performance of our proposed methods in comparison with existing methods, and further illustrate the applications in two clinical studies for aggressive prostate cancer.</p> (onlinelibrary.wiley.com)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling">
			  <li><a href="https://dx.doi.org/10.1017/S0950268822001947?rft_dat=source%3Ddrss">Estimation of the incubation period and generation time of SARS-CoV-2 Alpha and Delta variants from contact tracing data</a><p> 
Quantitative information on epidemiological quantities such as the incubation period and generation time of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants is scarce. We analysed a dataset collected during contact tracing activities in the province of Reggio Emilia, Italy, throughout 2021. We determined the distributions of the incubation period for the Alpha and Delta variants using information on negative polymerase chain reaction tests and the date of last exposure from 282 symptomatic cases. We estimated the distributions of the intrinsic generation time using a Bayesian inference approach applied to 9724 SARS-CoV-2 cases clustered in 3545 households where at least one secondary case was recorded. We estimated a mean incubation period of 4.9 days (95% credible intervals, CrI, 4.4–5.4) for Alpha and 4.5 days (95% CrI 4.0–5.0) for Delta. The intrinsic generation time was estimated to have a mean of 7.12 days (95% CrI 6.27–8.44) for Alpha and of 6.52 days (95% CrI 5.54–8.43) for Delta. The household serial interval was 2.43 days (95% CrI 2.29–2.58) for Alpha and 2.74 days (95% CrI 2.62–2.88) for Delta, and the estimated proportion of pre-symptomatic transmission was 48–51% for both variants. These results indicate limited differences in the incubation period and intrinsic generation time of SARS-CoV-2 variants Alpha and Delta compared to ancestral lineages. 
</p> (dx.doi.org)</li>
			  </div>
			<div class="filterDiv all save surveillance std viral">
			  <li><a href="https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-022-07972-5">Temporal discrepancies in “rapid” HIV testing: explaining misdiagnoses at the point-of-care in Zimbabwe</a><p>Rapid diagnostic tests have revolutionized the HIV response in low resource and high HIV prevalence settings. However, disconcerting levels of misdiagnosis at the point-of-care call for research into their roo...</p> (bmcinfectdis.biomedcentral.com)</li>
			  </div>
			<div class="filterDiv all save omics">
			  <li><a href="https://www.nature.com/articles/s41598-022-27281-w">A high-throughput single-particle imaging platform for antibody characterization and a novel competition assay for therapeutic antibodies</a><p></p> (www.nature.com)</li>
			  </div>
			<div class="filterDiv all save surveillance vector">
			  <li><a href="https://www.nature.com/articles/s41598-022-26878-5">National-scale simulation of human movement in a spatially coupled individual-based model of malaria in Burkina Faso</a><p></p> (www.nature.com)</li>
			  </div>
			<div class="filterDiv all save viral">
			  <li><a href="https://www.jwatch.org/na55677/2023/01/05/sars-cov-2-nucleic-acid-persists-multiple-organs-after">SARS-CoV-2 Nucleic Acid Persists in Multiple Organs After Acute Infection</a><p>In an autopsy study, researchers used highly sensitive assays to find viral RNA and antigens.</p> (www.jwatch.org)</li>
			  </div>
			<div class="filterDiv all save viral vaccines">
			  <li><a href="https://www.nature.com/articles/s41590-022-01393-x">The COVID-19 immunology masterclass enters its third year</a><p></p> (www.nature.com)</li>
			  </div>
			<div class="filterDiv all save fungal vaccines">
			  <li><a href="https://www.nature.com/articles/s41591-022-02110-9">T cell receptor repertoires associated with control and disease progression following Mycobacterium tuberculosis infection</a><p></p> (www.nature.com)</li>
			  </div>
			<div class="filterDiv all save viral vaccines">
			  <li><a href="https://www.jwatch.org/na55670/2023/01/04/early-look-real-world-experience-with-bivalent-covid-19">An Early Look at Real-World Experience with Bivalent COVID-19 Vaccines</a><p>Compared with monovalent mRNA vaccines, bivalent vaccines yield higher antibody titers against Omicron variants and appear to have higher effectiveness against COVID-19–related hospitalization.</p> (www.jwatch.org)</li>
			  </div>
			
		</ol>
		</div>
		<script>
		filterSelection("all")
		</script>
		

		<footer>
		    <p><a href="https://feed.wakeforestid.com/bibliography.bibtex" download>Download this bibliography</a></p>
		    <p><a href="https://github.com/wf-id/ideas-feed">What is this?</a></p>
			<p><a href="https://www.zotero.org/groups/4900647/ideas-feed/library">Zotero Library</a></p>
			<p><a href="https://www.researchrabbit.ai/">Research Rabbit</a></p>
			<p><a href="https://github.com/jamesroutley/news.routley.io">What is this based on</a></p>
			<p><a href="https://wakeforestid.com">Main Website</a></p>
		</footer>
	</body>
</html>
