
<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<title>IDEAS | Feed</title>
		<style>
			@import url("https://fonts.googleapis.com/css2?family=Nanum+Myeongjo&display=swap");

			body {
				font-family: "Nanum Myeongjo", serif;
				line-height: 1.7;
				max-width: 600px;
				margin: 50px auto 50px;
				padding: 0 12px 0;
				height: 100%;
			}

			li {
				padding-bottom: 16px;
			}
			.container {
				overflow: hidden;
			  }
			  
			  .filterDiv {
				float: left;
				width: 100%;
				text-align: left;
				margin: 2px;
				display: none; /* Hidden by default */
			  }
			  
			  /* The "show" class is added to the filtered elements */
			  .show {
				display: block;
			  }
			  
			  /* Style the buttons */
			  .btn {
				border: none;
				outline: none;
				padding: 12px 16px;
				background-color: #f1f1f1;
				cursor: pointer;
			  }
			  
			  /* Add a light grey background on mouse-over */
			  .btn:hover {
				background-color: #ddd;
			  }
			  
			  /* Add a dark background to the active button */
			  .btn.active {
				background-color: #666;
				color: white;
			  }
		</style>
		<script>
		filterSelection("all")
function filterSelection(c) {
  var x, i;
  x = document.getElementsByClassName("filterDiv");
  if (c == "all") c = "";
  // Add the "show" class (display:block) to the filtered elements, and remove the "show" class from the elements that are not selected
  for (i = 0; i < x.length; i++) {
    w3RemoveClass(x[i], "show");
    if (x[i].className.indexOf(c) > -1) w3AddClass(x[i], "show");
  }
}

// Show filtered elements
function w3AddClass(element, name) {
  var i, arr1, arr2;
  arr1 = element.className.split(" ");
  arr2 = name.split(" ");
  for (i = 0; i < arr2.length; i++) {
    if (arr1.indexOf(arr2[i]) == -1) {
      element.className += " " + arr2[i];
    }
  }
}

// Hide elements that are not selected
function w3RemoveClass(element, name) {
  var i, arr1, arr2;
  arr1 = element.className.split(" ");
  arr2 = name.split(" ");
  for (i = 0; i < arr2.length; i++) {
    while (arr1.indexOf(arr2[i]) > -1) {
      arr1.splice(arr1.indexOf(arr2[i]), 1);
    }
  }
  element.className = arr1.join(" ");
}

// Add active class to the current control button (highlight it)
var btnContainer = document.getElementById("myBtnContainer");
var btns = btnContainer.getElementsByClassName("btn");
for (var i = 0; i < btns.length; i++) {
  btns[i].addEventListener("click", function() {
    var current = document.getElementsByClassName("active");
    current[0].className = current[0].className.replace(" active", "");
    this.className += " active";
  });
}
		</script>
	</head>
	<body>
	<P></P><cite><a href="https://wakeforestid.com/">home</a></cite>
		<h1>Latest Literature</h1>
		<div id="myBtnContainer">
		<button class="btn active" id="al2l" onclick="filterSelection('all')"> Show all</button>
		<button class="btn" id="all" onclick="filterSelection('amr')"> AMR</button>
		<button class="btn" onclick="filterSelection('fungal')"> Fungal</button>
		<button class="btn" onclick="filterSelection('vector')"> Vector</button>
		<button class="btn" onclick="filterSelection('bacteria')"> Bacterial</button>
		<button class="btn" onclick="filterSelection('std')"> STD</button>
		<button class="btn" onclick="filterSelection('surveillance')"> Surveillance</button>
		<button class="btn" onclick="filterSelection('modelling')"> Modelling</button>
		<button class="btn" onclick="filterSelection('viral')"> Viral</button>
		<button class="btn" onclick="filterSelection('omics')"> Omics and Biomarkers</button>
		</div>

		<div class="container">
		<ol>
			<div class="filterDiv all save surveillance bacteria">
			  <li><a href="https://www.sciencedirect.com/science/article/pii/S1755436524000100?dgcid=rss_sd_all">Acquisition and Clearance Dynamics of <em>Campylobacter</em> spp. in Children in Low- and Middle-Income Countries</a><p>Publication date: Available online 10 February 2024Source: EpidemicsAuthor(s): Dehao Chen, Arie H. Havelaar, James A. Platts-Mills, Yang Yang</p> (www.sciencedirect.com)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling viral vaccines">
			  <li><a href="https://www.sciencedirect.com/science/article/pii/S1755436524000070?dgcid=rss_sd_all">Projecting Omicron scenarios in the US while tracking population-level immunity</a><p>Publication date: Available online 10 February 2024Source: EpidemicsAuthor(s): Anass Bouchnita, Kaiming Bi, Spencer J. Fox, Lauren Ancel Meyers</p> (www.sciencedirect.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria vaccines">
			  <li><a href="https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-024-08979-w">Prevalence of Cryptosporidium and Giardia infections in under-five children with diarrhoea in Blantyre, Malawi</a><p>Diarrhoeal diseases are common among children in low- and middle-income countries and are major causes of morbidity and mortality. Cryptosporidium and Giardia are considered to be the main parasitic causes of dia...</p> (bmcinfectdis.biomedcentral.com)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling">
			  <li><a href="https://www.sciencedirect.com/science/article/pii/S1755436524000033?dgcid=rss_sd_all">Differences between the true reproduction number and the apparent reproduction number of an epidemic time series</a><p>Publication date: March 2024Source: Epidemics, Volume 46Author(s): Oliver Eales, Steven Riley</p> (www.sciencedirect.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria modelling">
			  <li><a href="https://academic.oup.com/cid/article/78/Supplement_1/S1/7593852?rss=1">Tularemia: A Storied History, An Ongoing Threat</a><p>Assistant Secretary for Preparedness and Response</p> (academic.oup.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria modelling">
			  <li><a href="https://academic.oup.com/cid/article/78/Supplement_1/S4/7593859?rss=1">Antimicrobial Susceptibility of Francisella tularensis Isolates in the United States, 2009–2018</a><p>AbstractFrancisella tularensis is the causative agent of tularemia. We tested the susceptibility of 278 F. tularensis isolates from the United States received during 2009–2018 to 8 antimicrobial drugs (ciprofloxacin, levofloxacin, doxycycline, tetracycline, gentamicin, streptomycin, chloramphenicol, and erythromycin). All isolates were susceptible to all tested drugs.</p> (academic.oup.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria modelling">
			  <li><a href="https://academic.oup.com/cid/article/78/Supplement_1/S55/7593860?rss=1">Neuroinvasive Francisella tularensis Infection: Report of 2 Cases and Review of the Literature</a><p>AbstractBackgroundNeuroinvasive infection with Francisella tularensis, the causative agent of tularemia, is rare. Establishing clinical suspicion is challenging if risk factors or clinical features classically associated with tularemia are absent. Tularemia is treatable with antibiotics; however, there are limited data to inform management of potentially fatal neuroinvasive infection.MethodsWe collected epidemiologic and clinical data on 2 recent US cases of neuroinvasive F. tularensis infection, and performed a literature review of cases of neuroinvasive F. tularensis infection published after 1950.ResultsOne patient presented with focal neurologic deficits and brain lesions; broad-range molecular testing on resected brain tissue detected F. tularensis. The other patient presented with meningeal signs; tularemia was suspected based on animal exposure, and F. tularensis grew in cerebrospinal fluid (CSF) culture. Both patients received combination antibiotic therapy and recovered from infection. Among 16 published cases, tularemia was clinically suspected in 4 cases. CSF often displayed lymphocytic pleocytosis. Among cases with available data, CSF culture was positive in 13 of 16 cases, and F. tularensis antibodies were detected in 11 of 11 cases. Treatment typically included an aminoglycoside combined with either a tetracycline or a fluoroquinolone. Outcomes were generally favorable.ConclusionsClinicians should consider neuroinvasive F. tularensis infection in patients with meningitis and signs suggestive of tularemia or compatible exposures, lymphocyte-predominant CSF, unrevealing standard microbiologic workup, or lack of response to empiric bacterial meningitis treatment. Molecular testing, culture, and serologic testing can reveal the diagnosis. Favorable outcomes can be achieved with directed antibiotic treatment.</p> (academic.oup.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria modelling">
			  <li><a href="https://academic.oup.com/cid/article/78/Supplement_1/S7/7593854?rss=1">Efficacy of Doxycycline and Ciprofloxacin for Treatment of Pneumonic Tularemia in Cynomolgus Macaques</a><p>AbstractBackgroundThe incidence of pneumonic tularemia is very low; therefore, it is not feasible to conduct clinical efficacy testing of tularemia medical countermeasures (MCMs) in humans. The US Food and Drug Administration’s Animal Model Qualification Program under the Drug Development Tools Program is a regulatory pathway for animal models used in MCM efficacy testing and approval under the Animal Rule. The National Institute of Allergy and Infectious Diseases and Biomedical Advanced Research and Development Authority worked together to qualify the cynomolgus macaque model of pneumonic tularemia.MethodsUsing the model parameters and end points defined in the qualified model, efficacy of the antibiotics doxycycline and ciprofloxacin was evaluated in separate studies. Antibiotic administration, aimed to model approved human dosing, was initiated at time points of 24 hours or 48 hours after onset of fever as an indicator of disease.ResultsUpon aerosol exposure (target dose of 1000 colony-forming units) to Francisella tularensis SchuS4, 80% of vehicle-treated macaques succumbed or were euthanized. Ciprofloxacin treatment led to 10 of 10 animals surviving irrespective of treatment time. Doxycycline administered at 48 hours post-fever led to 10 of 10 animals surviving, while 9/10 animals survived in the group treated with doxycycline 24 hours after fever. Selected surviving animals in both the placebo and doxycycline 48-hour group showed residual live bacteria in peripheral tissues, while there were no bacteria in tissues from ciprofloxacin-treated macaques.ConclusionsBoth doxycycline and ciprofloxacin were efficacious in treatment of pneumonic tularemia, although clearance of bacteria may be different between the 2 drugs.</p> (academic.oup.com)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria modelling">
			  <li><a href="https://dx.doi.org/10.1017/S0950268824000116?rft_dat=source%3Ddrss">Geographical distribution of invasive meningococcal disease and carriage: A spatial analysis</a><p>Little information exists concerning the spatial relationship between invasive meningococcal disease (IMD) cases and Neisseria meningitidis (N. meningitidis) carriage. The aim of this study was to examine whether there is a relationship between IMD and asymptomatic oropharyngeal carriage of meningococci by spatial analysis to identify the distribution and patterns of cases and carriage in South Australia (SA). Carriage data geocoded to participants’ residential addresses and meningococcal case notifications using Postal Area (POA) centroids were used to analyse spatial distribution by disease- and non-disease-associated genogroups, as well as overall from 2017 to 2020. The majority of IMD cases were genogroup B with the overall highest incidence of cases reported in infants, young children, and adolescents. We found no clear spatial association between N. meningitidis carriage and IMD cases. However, analyses using carriage and case genogroups showed differences in the spatial distribution between metropolitan and regional areas. Regional areas had a higher rate of IMD cases and carriage prevalence. While no clear relationship between cases and carriage was evident in the spatial analysis, the higher rates of both carriage and disease in regional areas highlight the need to maintain high vaccine coverage outside of the well-resourced metropolitan area.</p> (dx.doi.org)</li>
			  </div>
			<div class="filterDiv all save surveillance modelling">
			  <li><a href="https://www.nature.com/articles/s41467-024-45027-2">Predicting the risk and speed of drug resistance emerging in soil-transmitted helminths during preventive chemotherapy</a><p></p> (www.nature.com)</li>
			  </div>
			<div class="filterDiv all save surveillance amr bacteria modelling omics">
			  <li><a href="https://www.nature.com/articles/s41467-023-42211-8">Inferring bacterial transmission dynamics using deep sequencing genomic surveillance data</a><p></p> (www.nature.com)</li>
			  </div>
			<div class="filterDiv all save surveillance vector viral">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/12/23-0587_article">Crimean-Congo Hemorrhagic Fever Virus Seropositivity among Dromedary Camels, Algeria, 2020–2021</a><p>K. Guidoum et al.</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save surveillance bacteria modelling">
			  <li><a href="https://www.sciencedirect.com/science/article/pii/S1755436523000610?dgcid=rss_sd_all">Modelling flock heterogeneity in the transmission of peste des petits ruminants virus and its impact on the effectiveness of vaccination for eradication</a><p>Publication date: December 2023Source: Epidemics, Volume 45Author(s): Bethan Savagar, Bryony A. Jones, Mark Arnold, Martin Walker, Guillaume Fournié</p> (www.sciencedirect.com)</li>
			  </div>
			<div class="filterDiv all save surveillance viral vaccines">
			  <li><a href="https://www.sciencedirect.com/science/article/pii/S1755436523000622?dgcid=rss_sd_all">Estimation of waning vaccine effectiveness from population-level surveillance data in multi-variant epidemics</a><p>Publication date: December 2023Source: Epidemics, Volume 45Author(s): Hiroaki Murayama, Akira Endo, Shouto Yonekura</p> (www.sciencedirect.com)</li>
			  </div>
			<div class="filterDiv all save surveillance viral vaccines">
			  <li><a href="https://academic.oup.com/jid/article/228/Supplement_6/S376/7320338?rss=1">Bunyavirales: Scientific Gaps and Prototype Pathogens for a Large and Diverse Group of Zoonotic Viruses</a><p>AbstractResearch directed at select prototype pathogens is part of the approach put forth by the National Institute of Allergy and Infectious Disease (NIAID) to prepare for future pandemics caused by emerging viruses. We were tasked with identifying suitable prototypes for four virus families of the Bunyavirales order (Phenuiviridae, Peribunyaviridae, Nairoviridae, and Hantaviridae). This is a challenge due to the breadth and diversity of these viral groups. While there are many differences among the Bunyavirales, they generally have complex ecological life cycles, segmented genomes, and cause a range of human clinical outcomes from mild to severe and even death. Here, we delineate potential prototype species that encompass the breadth of clinical outcomes of a given family, have existing reverse genetics tools or animal disease models, and can be amenable to a platform approach to vaccine testing. Suggested prototype pathogens outlined here can serve as a starting point for further discussions.</p> (academic.oup.com)</li>
			  </div>
			<div class="filterDiv all save std viral">
			  <li><a href="https://wwwnc.cdc.gov/eid/article/29/12/23-1146_article">Tecovirimat Resistance in Mpox Patients, United States, 2022–2023</a><p>T. G. Smith et al.</p> (wwwnc.cdc.gov)</li>
			  </div>
			<div class="filterDiv all save surveillance vector viral">
			  <li><a href="https://www.sciencedirect.com/science/article/pii/S1755436523000579?dgcid=rss_sd_all">Model-based estimates of chikungunya epidemiological parameters and outbreak risk from varied data types</a><p>Publication date: December 2023Source: Epidemics, Volume 45Author(s): Alexander D. Meyer, Sandra Mendoza Guerrero, Natalie E. Dean, Kathryn B. Anderson, Steven T. Stoddard, T. Alex Perkins</p> (www.sciencedirect.com)</li>
			  </div>
			
		</ol>
		</div>
		<script>
		filterSelection("all")
		</script>
		

		<footer>
		    <p><a href="https://feed.wakeforestid.com/bibliography.bibtex" download>Download this bibliography</a></p>
		    <p><a href="https://github.com/wf-id/ideas-feed">What is this?</a></p>
			<p><a href="https://www.zotero.org/groups/4900647/ideas-feed/library">Zotero Library</a></p>
			<p><a href="https://www.researchrabbit.ai/">Research Rabbit</a></p>
			<p><a href="https://github.com/jamesroutley/news.routley.io">What is this based on</a></p>
			<p><a href="https://wakeforestid.com">Main Website</a></p>
		</footer>
	</body>
</html>
